nixos/modules/tasks/filesystems.nix

```diff
@@ -239,6 +239,8 @@ in
       '';
     };
 
+    boot.initrd.gptAutoRoot.enable = lib.mkEnableOption (lib.mdDoc "systemd-gpt-auto-generator for the root file system");
```

Comment:
Missing "Use" to start with this sentence? And maybe:

> Use `systemd-gpt-auto-generator` to find the root file system. See the [Discoverable Partitions Spec](https://uapi-group.org/specifications/specs/discoverable_partitions_specification/) for more.

Reply:
`mkEnableOption` prefixes with `Whether to enable `